wdl/BatchEvidenceMerging.wdl

```diff
@@ -2,289 +2,187 @@ version 1.0
 
 import "Structs.wdl"
 
-workflow EvidenceMerging {
+workflow BatchEvidenceMerging {
   input {
     Array[String] samples
     Array[File?]? BAF_files
     Array[File] PE_files
     Array[File] SR_files
-    File inclusion_bed
-    File genome_file
+    Array[File] LD_files
+    File ld_locs_vcf
+    File reference_dict
     String batch
-
-    #Single file size estimates
-    Int? BAF_size_mb
-    Int? PE_size_mb
-    Int? SR_size_mb
-
-    Int? disk_overhead_gb   # Fixed extra disk
-    String sv_base_mini_docker
-    RuntimeAttr? runtime_attr_set_sample
-
-    RuntimeAttr? runtime_attr_shard_baf
-    RuntimeAttr? runtime_attr_merge_baf
-    RuntimeAttr? runtime_attr_shard_pe
-    RuntimeAttr? runtime_attr_merge_pe
-    RuntimeAttr? runtime_attr_shard_sr
-    RuntimeAttr? runtime_attr_merge_sr
-  }
-
-  Array[String] contigs = transpose(read_tsv(genome_file))[0]
-
-  if (defined(BAF_files)) {
-    Array[File?] BAF_files_value = select_first([BAF_files])
-    scatter (i in range(length(samples))) {
-      if (defined(BAF_files_value[i])) {
-        call SetSampleId as SetSampleIdBAF {
-           input:
-             file = select_first([BAF_files_value[i]]),
-             sample_id = samples[i],
-             output_name = "BAF_GatherBatchEvidence.~{samples[i]}.txt.gz",
-             sample_column_index = 4,
-             sv_base_mini_docker = sv_base_mini_docker,
-             runtime_attr_override = runtime_attr_set_sample
-        }
-      }
-    }
```

Comment:
Deleting these from the main pipeline is definitely the right move at this point. However, we still need a means of changing sample IDs in evidence files for some projects. Can you add a new `SetSampleIdLegacy` workflow that takes in an SR, PE, and BAF file for a single sample and converts the sample ID in each file?

Also can you update the `Sample ID requirements` section of the README and include a pointer to the `SetSampleIdLegacy` workflow for evidence files generated with previous versions of the pipeline?